packages/date-time/src/components/DatePicker/DatePicker.base.tsx

```diff
@@ -487,7 +489,7 @@ export const DatePickerBase = React.forwardRef(
             {...calloutProps}
             className={css(classNames.callout, calloutProps && calloutProps.className)}
             // eslint-disable-next-line react/jsx-no-bind
-            onDismiss={calendarDismissed}
+            onDismiss={calloutDismissed}
```

Comment:
unrelated bug fix that was annoying me- onDismiss accepts a function where the first parameter is a mouseEvent, but we were passing a function where the first parameter is an option Date. so the calendarDismissed function would get called with a MouseEvent instead of a Date and we'd get an error in the console